org.jousse.vincent.Pomodorolm.yml

```diff
@@ -0,0 +1,45 @@
+id: org.jousse.vincent.Pomodorolm
+
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+
+command: pomodorolm
+finish-args:
+  - --socket=wayland # Permission needed to show the window
+  - --socket=fallback-x11 # Permission needed to show the window
+  - --socket=pulseaudio # We want to be able to play sounds
+  - --device=dri # OpenGL
+  - --share=ipc
+  - --env=APPDIR=/app # Fake AppImage env for Tauri. Needed so than it will look for assets in /app
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # Required for the systray to work
+  - --own-name=org.kde.*
+  # Needed for the systray to work properly. libappindicator emits a dbus signal with the systray icon absolute path
+  # so flatpak and the host need to both have access to the icon for the systray to work properly
+  - --filesystem=xdg-data
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/vjousse/pomodorolm/releases/download/app-v0.1.8/pomodorolm_0.1.8_amd64.deb
+        sha256: ff1b5b6ed87489f446c852ef47e5bec1018347a371518bfe7411f932ecad88a4
```

Comment:
why not build from source?

Reply:
Building from the deb package is how the official Tauri doc recommends to it https://v2.tauri.app/distribute/flatpak/
Doing it in the flatpak manifest would require setting up rust and npm build tools. Is it something recommended?

Reply:
Thanks the pointers, I'll give it a try. I suppose that it will be a good opportunity to update the tauri documentation :+1: